gnomad/utils/annotations.py

```diff
@@ -2294,21 +2294,44 @@ def _create_group_dicts(
         "cohort": {"id": "ALL"},
     }
 
+    # Check allele balance for heterozygotes values.
+    # Flagged values are those in bins >0.9. Each is 0.05, so last 2.
+    if len(input_struct.ab_hist_alt.bin_freq) != 20:
+        raise ValueError(
+            f"{gnomad_id} ab_hist_alt.bin_freq had "
+            f"{len(input_struct.ab_hist_alt.bin_freq)} items, expected 20"
+        )
+    # The bin_freq should be in order but we can verify the order from bin_edges
+    ab_bin_freq = list(
+        map(
+            lambda tpl: tpl[1],
+            sorted(
+                zip(
+                    input_struct.ab_hist_alt.bin_edges,
+                    input_struct.ab_hist_alt.bin_freq,
+                ),
+                key=lambda tpl: tpl[0],
+            ),
+        )
+    )
+
     # Create ancillaryResults for additional frequency and popMaxFAF95 information
     ancillaryResults = {
         "homozygotes": overall_freq["homozygote_count"],
         "popMaxFAF95": {
             "frequency": input_struct.faf95.popmax,
             "confidenceInterval": 0.95,
         },
+        "qcFilters": list(input_struct.filters),
+        "lowComplexityRegion": input_struct.region_flag.lcr,
+        "heterozygousAlleleBalanceFlagged": sum(ab_bin_freq[-2:]),
```

Comment:
so this is just the count of how many genotypes were flagged?

Reply:
Yes the count of those over the 0.9 threshold. Let me know if this seems correct or if there are any weird cases that might need to be handled differently